code/plugins/api/articles/articles/helper/simpleschema.php

```
@@ -0,0 +1,167 @@
+use Joomla\CMS\Uri\Uri;
+//jimport( 'simpleschema.easyblog.blog.post' );
+require_once JPATH_SITE.'/plugins/api/articles/articles/blogs/blog/post.php';
+require_once JPATH_SITE.'/plugins/api/articles/articles/blogs/category.php';
+require_once JPATH_SITE.'/plugins/api/articles/articles/helper/contenthelper.php';
+
+//for world merit development
+//require_once JPATH_SITE.'/components/com_content/models/media.php';
+
+class BlogappSimpleSchema 
+{
+	public function mapPost($blog, $strip_tags='', $text_length=0, $skip=array())
```

PHPMD (2.15.0)

Avoid unused parameters such as '$strip_tags'. (Unused Code Rules)

(UnusedFormalParameter)